pkg/credentialprovider/aws/aws_credentials.go

```diff
@@ -48,7 +48,7 @@ var (
 
 // init registers a credential provider for each registryURLTemplate and creates
 // an ECR token getter factory with a new cache to store token getters
-func init() {
+func Register() {
```

Comment:
Comments need to be updated... is `Register` part of an interface somewhere?

Reply:
Updated, No it is not part of any interface. Introduced this to have collective initialisation at 
`pkg/credentialprovider/register/register.go`